woocommerce/woocommerce
# Playwright 1.33 and UI mode

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR updates Playwright to version 1.33 and introduces the UI mode command.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Confirm that tests pass on CI
2. Pull this branch
3. `pnpm env:start`
4. `pnpm test:e2e-pw` (you may have to run `pnpx playwright install` first -- you'll be prompted)
5. Confirm tests pass
6. `pnpm test:e2e-pw-ui`
7. Confirm that Playwright UI window is shown.
